libcudacxx/test/libcudacxx/CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ if (NOT LIBCUDACXX_TEST_WITH_NVRTC)
   # Build but don't run the tests. Used by CI to pre-seed sccache for the test machines.
   # Only executed if explicitly requested.
   add_custom_target(libcudacxx.test.lit.precompile
-    DEPENDS libcudacxx.test.public_headers libcudacxx.test.internal_headers libcudacxx.test.public_headers_host_only
+    DEPENDS libcxx.test.lit
```

Comment:
Why remove the other dependencies?

Reply:
The parent directory does `add_subdirectory(libcudacxx)` before `add_subdirectory(libcxx)`, this target won't be defined yet.